scripts/cola_baseline.sh

```diff
@@ -0,0 +1,25 @@
+# baseline experiments on cola.
+# don't actually run this sh file as a whole,
```

Comment:
Why not?

Reply:
Because it is srun, it will run one after another.
Now we have removed the prince related command, we don't need this line anymore.